libs/blocks/modal/modal.js

```diff
@@ -126,16 +126,20 @@ export async function getModal(el, custom) {
     }
   });
 
-  dialog.addEventListener('keydown', (event) => {
-    if (event.key === 'Escape') {
-      closeModals([dialog]);
-    }
-  });
 
   dialog.append(close, content);
   document.body.append(dialog);
   dialog.insertAdjacentElement('afterend', curtain);
-  firstFocusable.focus(focusVisible);
+  document.getElementsByClassName("dialog-close")[0]?.focus({ focusVisible: true });
```

Comment:
This is not ideal due to having multiple modals on page at a given time.

Isn't the close button created in this function already?

@JasonHowellSlavin didn't this already get addressed?

Reply:
Yes. When I did the focus trap. Close is not the expected default focus when there is content within a modal that contains focusable items. https://www.w3.org/WAI/ARIA/apg/example-index/dialog-modal/dialog

Reply:
@JasonHowellSlavin @auniverseaway I added default focus on close button as a part of requirement in ticket. https://jira.corp.adobe.com/browse/MWPW-123567  should this be changed  or removed from requirement as jason said it should not focus on close by default when there in content in modal.

Reply:
We clarified within comments on the ticket, so this should be removed based on WCAG/W3 a11y requirements.